packages/contracts-bedrock/src/L2/SuperchainTokenBridge.sol

```diff
@@ -86,7 +86,7 @@ contract SuperchainTokenBridge {
     function relayERC20(address _token, address _from, address _to, uint256 _amount) external {
         if (msg.sender != MESSENGER) revert Unauthorized();
 
-        (address crossDomainMessageSender, uint256 source) =
+        (address crossDomainMessageSender, uint256 source, /*address entrypoint*/, /*uint256 nonce*/ ) =
```

Comment:
No need to add those comments
```suggestion
        (address crossDomainMessageSender, uint256 source, ,  ) =
```

Reply:
Addressed [here](https://github.com/defi-wonderland/optimism/pull/159/commits/35307d13b4413742580babadb9ea6e9f50ed6e8c).